### PR TITLE
Update Analysis3D.py

### PR DESCRIPTION
**Pull request by PipKat**

### CODE REVIEW

**Comment by PipKat** [comment]
In the FieldAnalysis3D class, solutiontype (with no hyphen) is used. In the FieldAnalysis2D class (and in most other places), solution_type is used. I figured I'd point this out.

File: pyaedt/application/Analysis3D.py [hunk 1]
```diff
@@ -71,17 +98,18 @@ def components3d(self):
 
     @aedt_exception_handler
     def get_components3d_vars(self, component3dname):
-        """Read the a3dComp file and check for variables. It polulates a dictionary with default value
+        """Read the A3DCOMP flie  and check for variables. It polulates a dictionary with default value
```
Comment (PipKat) [inline pyaedt/application/Analysis3D.py:101]:
```suggestion
        """Read the A3DCOMP file and check for variables. 
```

File: pyaedt/application/Analysis3D.py [hunk 2]
```diff
@@ -158,26 +187,29 @@ def get_property_value(self, objectname, property, type=None):
...
+        no_vacuum : bool, optional
+            Whether to include vacuum objects for the copied objects. 
+            The default is ``True``.
         no_pec :
-            define if pec objects have to be copied (Default value = True)
+            Whether to include pec objects for the copied objects. The 
+            default is ``True``.
         include_sheets :
-            include sheets in the objects that have to be copied (Default value = False)
+            Whether to include sheets for the copied ojects. The
```
Comment (PipKat) [inline pyaedt/application/Analysis3D.py:205]:
```suggestion
            Whether to include sheets for the copied objects. The 
```

**Comment by akaszynski** [comment]
> In the FieldAnalysis3D class, solutiontype (with no hyphen) is used. In the FieldAnalysis2D class (and in most other places), solution_type is used. I figured I'd point this out.

That needs to be pointed out as an issue.  I know the issue list is getting quite long, but tracking these issues is important so we can track this and eventually solve them.

File: pyaedt/application/Analysis3D.py [hunk 3]
```diff
@@ -205,30 +237,32 @@ def copy_solid_bodies_from(self, design, object_list=None, no_vacuum=True, no_pe
...
     def get_all_sources(self):
-        """Get list of all setup sources.
+        """Retrieve all setup sources.
 
         Returns
         -------
         list
-            All setup sources.
+            List of setup sources.
+
```
Comment (akaszynski) [inline pyaedt/application/Analysis3D.py:246]:
```suggestion

```

File: pyaedt/application/Analysis3D.py [hunk 4]
```diff
@@ -205,30 +237,32 @@ def copy_solid_bodies_from(self, design, object_list=None, no_vacuum=True, no_pe
...
+        """Set the source context.
 
         Parameters
         ----------
         sources : list
-            Source names.
+            List of source names.
         number_of_modes : int, optional
-            Number of modes, by default 1
+            Number of modes. The  default is `1`.
```
Comment (akaszynski) [inline pyaedt/application/Analysis3D.py:259]:
```suggestion
            Number of modes. The  default is ``1``.
```

**Review by akaszynski — APPROVED** [review]
Approved with minor suggestions.

**Pull request merged**